components/button/src/button/button.js

```diff
@@ -37,6 +37,14 @@ export const Button = ({
         }
     }, [initialFocus, ref.current])
 
+    const { 'aria-label': ariaLabel, title } = otherProps
+
+    if (!children && !title && !ariaLabel) {
+        console.debug(
+            'Button component has no children but is missing title or ariaLabel attribute.'
```

Comment:
Log statement here does not match the if-condition above. Here you need both `title` and `ariaLabel` to be present.

Do we want to `ariaLabel` to fallback to `title`?